examples/events/snowplow-debugger/search-input.json

```diff
@@ -11,15 +12,17 @@
         ]
       },
       "queryTypes": {
-        "type": "array"
+        "type": "array",
+        "minLength": 1
       },
       "searchRequestId": {
         "type": "string",
         "maxLength": 256,
         "minLength": 1
       },
       "query": {
-        "type": "string"
+        "type": "string",
+        "minLength": 1
```

Comment:
I think this maps to `phrase` in `productSearch` request. If so, there are cases where it could be empty (when browsing a category page, gql query filters on a category but no phrase is provided).